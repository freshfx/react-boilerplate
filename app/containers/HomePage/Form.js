import styled from '@emotion/styled'

const Form = styled.form`
  margin-bottom: 1em;
`

export default Form
