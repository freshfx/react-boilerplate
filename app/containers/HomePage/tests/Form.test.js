import 'jest-styled-components'
import React from 'react'
import {mount} from 'enzyme'

import Form from '../Form'

const renderComponent = (props = {}) => mount(<Form {...props} />)

describe('<Form />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderComponent()
    expect(renderedComponent).toMatchSnapshot()
  })
})

