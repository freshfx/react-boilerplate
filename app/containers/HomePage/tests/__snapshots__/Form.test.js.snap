// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should match snapshot 1`] = `
.emotion-0 {
  margin-bottom: 1em;
}

<form
  class="emotion-0 emotion-1"
/>
`;
